back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/import_ideas_service.rb

```diff
@@ -208,23 +214,27 @@ def add_location(idea_row, idea_attributes)
     end
 
     def add_phase(idea_row, idea_attributes)
-      return if idea_row[:phase_rank].blank?
+      if idea_row[:phase_id]
+        phase = Phase.find(idea_row[:phase_id])
+        idea_attributes[:creation_phase_id] = phase.id if phase&.native_survey?
```

Comment:
The check `phase&.native_survey?` should move to the participation context.

Reply:
Same question here, what do you mean exactly with 'move to the participation context'?